demo/build.gradle

```diff
@@ -160,7 +160,16 @@ dependencies {
     implementation 'com.google.android.material:material:1.0.0'
     implementation 'androidx.multidex:multidex:2.0.1'
     implementation 'com.android.support:multidex:1.0.3'
+
+    // Import library from project
     implementation project(':MuxExoPlayer')
+
+    // Example for how to import from maven instead for the 2.13.1 configuration
+//    implementation 'com.mux.stats.sdk.muxstats:MuxExoPlayer:dev-nb-jfrog-93fc9c3:r2.13.1@aar'
```

Comment:
should we change this example to not a dev build, but rather a production release?

Reply:
Good catch, but we are in a catch-22 because only this PR will create a production release to point at!

Reply:
Updated to reference the upcoming 2.5.0 release